lib/activegraphql/query.rb

```diff
@@ -4,19 +4,25 @@
 
 module ActiveGraphQL
   class Query < Support::Fancy
-    attr_accessor :url, :action, :params, :graph, :response
+    attr_accessor :url, :action, :params, :locale, :graph, :response
```

Comment:
My idea was to encapsulate the strategy to send a locale inside this `Query` class, what IMO is part of it's scope (as by now it is the one having the logic to perform requests). I want to avoid consumers (ie. the fetcher)  having to deal with that. 
So my point is: You create a query instance with some locale, and just call get. Then the query instance is the one responsible to know the way to send this locale (by now it's a header, but in the future could end up in other strategies... will see).

Summarising, this implementation is not for setting up headers configuration, but for providing a query with the ability to get content based on locale (what includes adding some headers to the request options).